src/main/kotlin/calculator/InputParser.kt

```diff
@@ -6,18 +6,21 @@ import calculator.exceptions.InvalidCharacterException
 import calculator.exceptions.NotEnoughElementsException
 
 object InputParser {
+    private val charactersRegex = Regex("[^0-9+\\-*/\\s]")
```

Comment:
I see this as a constant, and if it were one, I'd probably capitalize it. What do you think?